intSet/intSet/intSet.java

```diff
@@ -0,0 +1,190 @@
+package main;
+import java.util.Arrays;
+public class intSet 
+{    
+    static final int CONSTANT=1000;
+    static final private int universalSet[]=new int[CONSTANT];
+    /**
+     * This static block helps to initialize
+     */
+    static
+    {
+        for(int i=0;i<universalSet.length;i++)
+        {
+            universalSet[i]=i+1;
+        }
+    }
+    private int[] arraySet;
+    /**
+     * Parameterized Constructor used to initialize the input set by the user 
+     * @param inputSet denotes the set which the user will initialize
+     */
+    public intSet(int inputSet[]) 
+    {
+        try
+        {
+            validateSet(inputSet);
+        }
+        catch(AssertionError e)
+        {
+            System.out.println(e.getMessage());
+        }
+        this.arraySet=inputSet;
+    }
+    /**
+     * Used to validate the input
+     * @param inputSet denotes the set entered by the user
+     */
+    private void validateSet(int[] inputSet)
+    {
+        Arrays.sort(inputSet);
+        for(int i=0;i<inputSet.length;i++)
+        {
+            if(!(inputSet[i]>=1 && inputSet[i]<=1000))
+            {
+                throw new AssertionError("Elements not in range");
+            }
+            else if(inputSet.length<=0)
+            {
+                throw new AssertionError("Array is empty");
+            }
+        }
+        for(int i=0;i<inputSet.length-1;i++)
+        {
+            if(inputSet[i]==inputSet[i+1])
+            {
+                throw new AssertionError("Elements in the set are repeated");
+            }
+        }
+    }
+    /**
+     * This method is used to check whether x is an element of the universal set or not
+     * @param x denotes the element which is to be checked
+     * @return true if element is present in the set else false
+     */
+    public boolean isMember(int x)
+    {
+        for(int i=0;i<arraySet.length;i++)
+        {
+            if(arraySet[i]==x)
+                return true;
+        }
+        return false;
+    }
+    /**
+     * Used to return the size of the set entered by the user
+     * @return the size of the set entered by the user
+     */
+    public int size()
+    {
+        return arraySet.length;
+    }
+    /**
+     * Used to check whether s is a subset of the set entered by the user
+     * @param s denotes the set which is to be checked that whether it is a subset or not
+     * @return true if it is a subset else return false
+     */
+    public boolean isSubSet(intSet s)
+    {
+        boolean flag=true;
+        for(int i=0;i<s.arraySet.length;i++)
+        {
+            flag=true;
+            for(int j=0;j<this.arraySet.length;j++)
+            {
+                if(s.arraySet[i]!=this.arraySet[j])
+                {
+                    flag=false;
+                }
+                else
+                {
+                    flag=true;
+                    break;
+                }
+            }

```

Comment:
Can't we use isMember() here?

Reply:
Ma'am, isMember  method compares the element by the universal set but in subset we have to compare it with the set entered by the user

Reply:
ok